src/Link.js

```diff
@@ -27,8 +27,16 @@ class Link extends Component {
     this.onClick = this.onClick.bind(this);
   }
 
+  isAbsoluteURL(url) {
+    return url.startsWith("http://")
```

Comment:
Also I think the `.href` property on a link DOM node will explode out relative paths to the full URL. If thats true in all browsers / cases than this will never be false.